### PR TITLE
fix: vcDoubleClick and revealAllSpoilers patch

### PR DESCRIPTION
**Pull request by D3SOX**
~~Currently, that's the only `onClick` in that module. I could add back the `className: "channelMention"` check if so desired.~~
See second commit

### CODE REVIEW

**Comment by Nuckyz** [comment]
I am pretty sure the match and replace was correct, the issue was the find

File: src/plugins/vcDoubleClick.ts [hunk 1]
```diff
@@ -48,10 +48,10 @@ export default definePlugin({
         },
         {
             // channel mentions
-            find: ".EMOJI_IN_MESSAGE_HOVER",
+            find: ".shouldCloseDefaultModals",
```
Comment (Nuckyz) [inline src/plugins/vcDoubleClick.ts:51]:
```suggestion
            find: "[[2*",
```

**Comment by D3SOX** [comment]
> I am pretty sure the match and replace was correct, the issue was the find 

Oh didn't put that into consideration I just recreated it from scratch and that is what I've come up with

File: src/plugins/vcDoubleClick.ts (hunk 1 above)
Comment (D3SOX) [reply src/plugins/vcDoubleClick.ts:51]: `[[2*` doesn't work for me. That targets a different module. On Discord `Stable 175517 (1fc3105)`

**Comment by D3SOX** [comment]
@Nuckyz Okay it only works with how it's currently pushed for me. Can you double check and send your Discord version?

**Review by Nuckyz — APPROVED** [review]

**Comment by D3SOX** [comment]
Ven merge this pls

**Comment by Vendicated** [comment]
thank you!

**Pull request merged**